Doc/library/functions.rst

```diff
@@ -1220,9 +1220,9 @@ are always available.  They are listed here in alphabetical order.
       Added the *strict* parameter.
 
 
-.. function:: max(iterable, *, key=None)
-              max(iterable, *, default, key=None)
-              max(arg1, arg2, *args, key=None)
+.. function:: max(iterable, /, *, key=None)
+              max(iterable, /, *, default, key=None)
+              max(arg1, arg2, /, *args, key=None)
```

Comment:
Would be nice to also update the docstrings.

https://github.com/python/cpython/blob/26ae05e95c7c5f1a646e8ec7fa690c0e7b4ab8b9/Python/bltinmodule.c#L2038-L2045

Reply:
Resolving this as this was reverted as suggested above by @AA-Turner